src/player.js

```diff
@@ -35,10 +54,39 @@ export default class Player {
     let audio = document.querySelector("audio");
     input.value = audio.volume;
 
-    input.addEventListener("input", this.boundVolume);
+    return input;
+  }
 
-    let inlineplayer = document.querySelector("div.inline_player");
-    if (!inlineplayer.classList.contains("hidden")) inlineplayer.append(input);
+  static transferPlayButton() {
```

Comment:
@emcniece yeah, I was thinking about that when I was writing it. I think it's definitely something for the future (probably adding to the `utils.js`)

I am not sure if keeping the `td` tag would be very clean. If it's not in a table I feel like it could be confusing (and maybe might not act as expected?)

Reply:
Ah, agreed, I meant leave the `td` where it is and move the contents of that element. The `td` cell provides positioning styles (which might be part of the reason you want to get rid of it) but I think it'll collapse nicely when it's empty. An inner `div` wrapper without styling could provide an easy handle for moving the content.